docker/compose/docker-compose_hadoop284_hive233_spark353_amd64.yml

```diff
@@ -37,11 +36,12 @@ services:
       retries: 3
 
   datanode1:
-    image: apachehudi/hudi-hadoop_2.8.4-datanode:latest
+    image: apachehudi/hudi-hadoop_2.8.4-datanode:bullseye
```

Comment:
The image tag will be reverted from `bullseye` to `latest` once the PR is close to merging.